sdks/python/apache_beam/runners/portability/universal_local_runner_test.py

```diff
@@ -97,6 +97,22 @@ def raise_error(x):
          | 'StageC' >> beam.Map(raise_error)
          | 'StageD' >> beam.Map(lambda x: x))
 
+  def test_errors_traceback(self):
+    # TODO: figure out a way for runner to parse and raise the
```

Comment:
Does the test cover anything that test_erros_stage does not?

Reply:
No, it doesn't. I replaced this with raising `unittest.SkipTest`.